inventory/digitalocean/inventory

```diff
@@ -17,5 +17,6 @@ mesos_masters
 mesos_masters
 
 [weave_servers:children]
+mesos_masters
```

Comment:
If we'd use weave connect instead of launch, could we decouple the role form the inventory?